docs/release-guide.md

```diff
@@ -73,30 +73,59 @@ For details look at `.gitlab-ci.yml`
 
 ## MacOS
 
-Ideally, directly after the tag is created, start with the MacOS release. This has to be done manually, for now. There is a script for this:
+Ideally, directly after the tag is created, start with the MacOS release. As the binaries of x86/arm64 are not compatible with each other, we need to build on two MacOS architectures.This has to be done manually, for now. There is a script for this. Start with the build on x86:
+
+### MacOS x64 build
+
+```bash
+./utils/build-osx.sh  --version v2.0.5-pre4 --appleid "Satoshi Nakamoto (appleid)" --mail "satoshi@gmx.com" specterd package upload
+```
+
+This will create three artifacts on github:
+* specterd-v2.0.5-pre4-osx_x64.zip
+* SHA256SUMS-macos_x64
+* SHA256SUMS-macos_x64.asc
+
+### MacOS arm64 build
+
+The electron application will get built on the arm architecture. As it needs to store the sha256 hash in the electron-app, the make-hash target
+will not only hash the specterd but also download the other specterd and hash it.
 
 ```bash
-./utils/build-osx.sh  --version v1.13.1 --appleid "Satoshi Nakamoto (appleid)" --mail "satoshi@gmx.com" make-hash specterd electron sign upload
+./utils/build-osx.sh  --version v2.0.5-pre4 --appleid "Satoshi Nakamoto (appleid)" --mail "satoshi@gmx.com" specterd make-hash electron sign package upload
 ```
 
-This script also runs `github.py upload `, so two more binares and the hash and signature files are uploaded to GitHub:
+This will create three artifacts on github:
```

Comment:
```suggestion
This will create four artifacts on github:
```

Reply:
fixed in [b8a3a7c](https://github.com/cryptoadvance/specter-desktop/pull/2446/commits/b8a3a7ce4ef1e1dfedd24e856d765754f1d5843f)